etc/scylla-manager/scylla-manager.yaml

```diff
@@ -0,0 +1,157 @@
+# Scylla Manager config YAML
+
```

Comment:
I think this file is not needed ?
`/etc/scylla-manager/scylla-manager.yaml` is the configuration of scylla-manager server working on host.
Is it some leftover ?